scripts/build.sh

```diff
@@ -68,16 +65,22 @@ ubuntu_setup () {
 
     if [ $ARCH = "aarch64" ]
     then
-	echo "Detected ARM64 Linux!"
-	# TODO: We should start using container builds in actions!
-	SOLVER_FLAGS="$SOLVER_FLAGS -DENABLE_Z3=On -DZ3_DIR=/usr -DENABLE_GOTO_CONTRACTOR=OFF"
-	return
+        echo "Detected ARM64 Linux!"
+        # TODO: We should start using container builds in actions!
+        SOLVER_FLAGS="$SOLVER_FLAGS \
+            -DENABLE_Z3=On -DZ3_DIR=/usr \
+            -DENABLE_GOTO_CONTRACTOR=OFF \
+        "
+        return
     fi
-    
+
     sudo apt-get update &&
     sudo apt-get install -y $PKGS &&
+
     echo "Installing Python dependencies" &&
     pip3 install --user meson ast2json &&
+    pip3 install --user pyparsing toml &&
+    pip3 install --user pyparsing tomli &&
```

Comment:
Given that we already install `python-toml` as set in PKGS above and that there also is `python-tomli` as an Ubuntu package, is it necessary to "pip install" them?

Reply:
`python3-pyparsing` is available as an Ubuntu package as well. I would put all three into the PKGS list.